### ISSUE TITLE
Frontend: Move legend from left side of screen to the right side. 

### ISSUE THREAD

**Issue by maxgraziano** [chicommons/maps#261]
Reporter: Steve

Move legend from left side of screen to the right side. Needs to be pulled and tested on Master.

**Comment by oldmannseamus** [comment]
My understanding is that control position must be defined on the map implementation itself. I wasn't able to locate the map in this repo, as it's being rendered directly from OpenStreetMap via an iframe.

The legend/controls would need to be adjusted there.

**Comment by SteveEdiger** [comment]
/umap/static/umap/css/panel.css

Left and right are both defined...now find what's setting it.

Can't figure out what's setting it as panel.left.on. panel.right.on is in the CSS, but it's getting set somewhere I haven't found.

Perhaps we can override it [here:](https://github.com/umap-project/umap/blob/master/docs/config/customize.md)

**Comment by oldmannseamus** [comment]
I don't believe those settings are within this repo. No code references the `leaflet` library, and the only reference to `umap` is in the url of the iframe that holds the map. I couldn't find any reference in the umap docs that allows this to be customized through the querystring.

Can you provide access to the repo containing the map configuration itself? I might be able to give a more useful answer there.

**Comment by SteveEdiger** [comment]
We'll use a work-around to move the data extent and origin to the right so that the legend does not cover them.

**Comment by SteveEdiger** [comment]
Completed in master.
